### DISCUSSION TITLE
How to number figures without additional caption text?

### DISCUSSION THREAD

**Discussion by werkstattcodes** [quarto-dev/quarto-cli#2546] [Q&A]
I would like to number my figures without any additional caption text, just Figure 1, Figure 2...
When I use e.g.

```
#| fig-cap: ""
#| label: fig-chapter_d
```

I obtain e.g. `Figure 1: ?(caption)` as a result. I get the same result also when skipping the fig-cap row entirely from the chunk options. Is there any way to get the desired result?

Many thanks.

**Comment by cderv** [comment]
@dragonstyle it seems it is not possible to have empty caption, is it ? 

Maybe we should allow to only have `Figure 1` and no caption ?

**Reply by dragonstyle** [reply]
Currently it does seem that we require caption text (we define a figure today as having both a label with `fig-` prefix and a caption). It would be nice to support empty captions, though we will have to consider how to best support this (e.g. what is the syntax, can the user just omit the caption or must it be specified as empty, etc...). Given the work plan for 1.3, I think this would line up nicely there.

**Reply by cderv** [reply]
Ok. i'll transform this to an issue.

**Reply by cderv** [reply]
@werkstattcodes please do follow #2551 about this now. 

Thanks a lot for the suggestion

**Reply by werkstattcodes** [reply]
Great. Many thanks for the swift response!
